src/workspace-element.coffee

```diff
@@ -95,6 +95,50 @@ class WorkspaceElement extends HTMLElement
 
   focusPaneViewOnRight: -> @paneContainer.focusPaneViewOnRight()
 
+  copyItemToPaneOnLeft: (destroyOriginal) ->
```

Comment:
One more comment... all these methods belong in `PaneContainerElement` and you should just delegate to them from `WorkspaceElement` like we do with `focusPaneViewOn*` methods.